test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -135,4 +142,19 @@ public void getAllPersons() throws Exception {
 
         assertTrue(isIdentical(allPersons, personsToCheck));
     }
+
+    @Test
+    public void nameIsSimilar() throws Exception {
+        Name variationOne = new Name("John K Smith");
+        Name variationTwo = new Name("John K SMIth");
+        Name variationThree = new Name("John Smith");
+//        Name variationFour = new Name("Smith, John K");
```

Comment:
Avoid commenting code. Delete unused code. You can go back to a previous version, precisely our reason to do version control.